sdk/textanalytics/azure-ai-textanalytics/samples/sample_recognize_entities.py

```diff
@@ -45,42 +45,10 @@ def recognize_entities(self):
             print("\nDocument text: {}".format(documents[idx]))
             for entity in doc.entities:
                 print("Entity: \t", entity.text, "\tCategory: \t", entity.category,
-                      "\tConfidence Score: \t", round(entity.score, 3))
+                      "\tConfidence Score: \t", entity.score)
```

Comment:
In some places we are printing `Confidence Score` and in others `Score`. Could we be consistent?
Note: Analyze Sentiment has to say `Confidence Score`